Packs/Linkshadow/Integrations/Linkshadow/Linkshadow.yml

```diff
@@ -0,0 +1,146 @@
+commonfields:
+  id: Linkshadow - XSOAR
+  version: -1
+name: Linkshadow - XSOAR
+display: Linkshadow - XSOAR
+category: Data Enrichment & Threat Intelligence
+description: Fetch Network Anomalies data from LinkShadow and execute the remediation
+  Actions.
+detaileddescription: |2
+
+  ## Linkshadow
+  - This section explains how to configure the instance of Linkshadow in Cortex XSOAR.
+  - You can use the following API Key: `VxVQklA6O4r5B2KSS1phb3oBav6Lo3HUZONS2q6u`
+  - api_username : `lsadmin`
+  - plugin_id :  `xsoar_integration_1604211382`
+  - action : `fetch_entity_anomalies`
+
+  ---
+configuration:
+- display: API Key
+  name: apiKey
+  type: 4
+  required: true
+- display: URL
+  name: url
+  type: 0
+  required: true
+- display: ACTION
```

Comment:
If there's a list of possible actions, I recommend defining a list (type 15).
https://xsoar.pan.dev/docs/integrations/yaml-file